service/src/main/java/uk/gov/hmcts/reform/fpl/model/SupportingEvidenceBundle.java

```diff
@@ -17,7 +17,7 @@
 public class SupportingEvidenceBundle {
     private final String name;
     private final String notes;
-    @PastOrPresent(message = "Date of time received cannot be in the future")
+    @DateNotInFuture()
```

Comment:
Should custom message be used here, default one in not really precise, should tell e.g what date must not be in future

Reply:
will use custom message